cmd/commands/system.go

```diff
@@ -79,7 +80,7 @@ knative:
 		},
 	}
 
-	command.Flags().StringVarP(&options.Manifest, "manifest", "m", "", "file path of a manifest file referring to the YAML files to be applied")
+	command.Flags().StringVarP(&options.Manifest, "manifest", "m", "default", "file path of a manifest file referring to the YAML files to be applied")
```

Comment:
should we document 'latest' as an option?

Reply:
Eventually. My thought was to keep it slightly hidden for now until we feel comfortable with Istio's ability to be applied without issues.